include/tester/db/Database.hpp

```diff
@@ -14,114 +15,35 @@
 #include "mongocxx/uri.hpp"
 #include "bsoncxx/builder/stream/array.hpp"
 
-const std::string uri_str = "mongodb://192.168.1.123:27017";
-
 class Database
 {
 public:
     Database(const std::string &domain):
         _instance{},
-        _uri(mongocxx::uri(uri_str)),
+        _uri(mongocxx::uri(MONGO + domain + PORT)),
         _client(mongocxx::client(_uri)),
         _database(_client[DATABASE_NAME])
     {     
     }
 
     ~Database() = default;
 
-    void save(const std::string &collaction_name ,const bsoncxx::document::value &document)
+    void save(std::string collectionName ,const bsoncxx::document::value &document)
```

Comment:
I believe that the collection name should be a `static constexpr` variable in this file instead of being a parameter.

Reply:
done